draft-ietf-capport-architecture.xml

```diff
@@ -209,17 +205,16 @@
        <t>Captive Portal Conditions: site-specific requirements that a user
           or device must satisfy in order to gain access to the wider network.</t>
        <t>Captive Portal Enforcement: The network equipment which enforces the
-          traffic restriction and notifies the User Equipment it is in a captive
-          state.</t>
+          traffic restriction.</t>
        <t>Captive Portal User Equipment: Also known as User Equipment. A device
           which has voluntarily joined a network for purposes of communicating
           beyond the constraints of the captive network.</t>
        <t>Captive Portal Server: The web server providing a user interface for
           assisting the user in satisfying the conditions to escape
           captivity.</t>
        <t>Captive Portal Signaling Protocol: Also known as Signaling Protocol.
-          A protocol used by the Enforcement device to signal the User Equipment
-          information about the state of its captivity.
+          A protocol used to signal the User Equipment information about the
```

Comment:
Question: Is the signaling protocol really signaling information about "the state of captivity", or is it instead signaling about connection/transmission failures due to captivity enforcement? All it provides a hint about is that something is restricted or enforced/blocked, but can't really provide trustworthy information about other state (unless we design a different protocol).

Reply:
My understanding was that the signal existed only to inform the UE that it should check back with the API.

Reply:
I think we properly capture that now.